.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+
+node_js:
+  - 6
+
+before_script:
+  - npm run lint
```

Comment:
![](https://cbslive105.files.wordpress.com/2014/11/kzthtse.gif)

New commit pushed, let's see if this [Travis-CI builds](https://travis-ci.org/mozilla/fathom/builds/130674011) starts sometime soon.